web/packages/operations/src/transfer_for_kusama.ts

```diff
@@ -0,0 +1,148 @@
+import cron from "node-cron"
+import {cryptoWaitReady} from "@polkadot/util-crypto"
+import {fetchRegistry} from "./registry";
+import {Direction} from "@snowbridge/api/dist/forKusama";
+
+export const transferForKusama = async (transferName: string, direction: Direction, amount: bigint, tokenName: string) => {
+    let env = "local_e2e"
+    if (process.env.NODE_ENV !== undefined) {
+        env = process.env.NODE_ENV
+    }
+    const snowbridgeEnv = environment.SNOWBRIDGE_ENV[env]
+    if (snowbridgeEnv === undefined) {
+        throw Error(`Unknown environment '${env}'`)
+    }
+
+    const { config, kusamaConfig, ethChainId, name } = snowbridgeEnv
+    await cryptoWaitReady()
+
+    if (!kusamaConfig) {
+        throw Error(`Kusama config should be set`)
+    }
+
+    const kusamaParachains: { [paraId: string]: string } = {}
+    kusamaParachains[kusamaConfig?.BRIDGE_HUB_PARAID.toString()] = kusamaConfig?.PARACHAINS[config.BRIDGE_HUB_PARAID.toString()]
+    kusamaParachains[kusamaConfig?.ASSET_HUB_PARAID.toString()] = kusamaConfig?.PARACHAINS[config.ASSET_HUB_PARAID.toString()]
+
+    const ethApikey = process.env.REACT_APP_INFURA_KEY || ""
+    const ethChains: { [ethChainId: string]: string } = {}
+    Object.keys(config.ETHEREUM_CHAINS).forEach(
+        (ethChainId) =>
+            (ethChains[ethChainId.toString()] = config.ETHEREUM_CHAINS[ethChainId](ethApikey))
+    )
+    const context = new Context({
+        environment: name,
+        ethereum: {
+            ethChainId,
+            ethChains,
+            beacon_url: process.env["BEACON_NODE_URL"] || config.BEACON_HTTP_API,
+        },
+        polkadot: {
+            assetHubParaId: config.ASSET_HUB_PARAID,
+            bridgeHubParaId: config.BRIDGE_HUB_PARAID,
+            parachains: config.PARACHAINS,
+            relaychain: process.env["RELAY_CHAIN_URL"] || config.RELAY_CHAIN_URL,
+        },
+        kusama: {
+            assetHubParaId: kusamaConfig.ASSET_HUB_PARAID,
+            bridgeHubParaId: kusamaConfig.BRIDGE_HUB_PARAID,
+            parachains: kusamaParachains,
+        },
+        appContracts: {
+            gateway: config.GATEWAY_CONTRACT,
+            beefy: config.BEEFY_CONTRACT,
+        },
+    })
+
+    const [polkadotAssetHub, kusamaAssetHub] = await Promise.all([
+        context.assetHub(),
+        context.kusamaAssetHub(),
+    ])
+
+    if (!kusamaAssetHub) {
+        throw Error(`Kusama asset hub could not connect`)
+    }
+
+    const polkadot_keyring = new Keyring({ type: "sr25519" })
+
+    const SUBSTRATE_ACCOUNT = process.env["SUBSTRATE_KEY"]
+        ? polkadot_keyring.addFromUri(process.env["SUBSTRATE_KEY"])
+        : polkadot_keyring.addFromUri("//Ferdie")
+
+    const registry = await fetchRegistry(env, context)
+
+    let sourceAccountHex = "0x460411e07f93dc4bc2b3a6cb67dad89ca26e8a54054d13916f74c982595c2e0e";
+    let beneficiaryAccountHex = "0x460411e07f93dc4bc2b3a6cb67dad89ca26e8a54054d13916f74c982595c2e0e";
```

Comment:
I'd suggest not hardcoding the account here. Instead, consider loading it from an env variable so others can test with their own accounts.

Reply:
Added in https://github.com/Snowfork/snowbridge/pull/1463/commits/a0bca9365d718e6cfeae8bc7e2b110ed0ce5008b.